### PR TITLE
new svg icon for articleList

### PR DESCRIPTION
**Pull request by kcj002**
Adds new icon for list-alt-2 svg for use in widget edit form.

Not sure if warrants version bump. Plz send help.

### CODE REVIEW

File: src/components/Icon/icons/ArticleList.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import iconColorsMap from './colors';
+
+const ArticleList = ({ fill, ...props }) => (
+  <svg width="24" height="24" viewBox="0 0 24 24" fill={fill} {...props}>
```
Comment (mdespuits) [inline src/components/Icon/icons/ArticleList.js:6]: For all of our SVGs, we have run them through `svgo` to optimize every SVG in the library. One tool you can quickly use for this ishttps://jakearchibald.github.io/svgomg. You also need to include the `xmlns="http://www.w3.org/2000/svg"` attribute for consistency.

**Comment by kcj002** [comment]
> I'm not entirely sure that this is a good fit for RoverUI since this is domain-specific. I would lean in one of two directions:
> 
> * Add this component to the FE app
> * Add this as an icon to tk-ui.
> 
> @pixelbandito any thoughts on this?

That gets complicated for use in the front end where we have constants for the icons that are all coming from rover. I could change the name of the icon so that it is less TK specific of a name.

**Comment by mdespuits** [comment]
> I could change the name of the icon so that it is less TK specific of a name.

That seems like a reasonable thought.Maybe something like `mediaList`? I also am not completely opposed to the name `articleList` either. Just jumped out at me as a very Cision domain

File: src/components/Icon/icons/MediaList.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import iconColorsMap from './colors';
 
-const ArticleList = ({ fill, ...props }) => (
+const ListAlt2 = ({ fill, ...props }) => (
```
Comment (mdespuits) [inline src/components/Icon/icons/MediaList.js:5]: If you rename this to `MediaList`, the const should match the file name 😄

Comment (kcj002) [reply src/components/Icon/icons/MediaList.js:5]: ugh. Had the change but didn't save/commit it. it's not my day. 🤦‍♀️

**Comment by pixelbandito** [comment]
Hella tight

**Pull request merged**